### ISSUE TITLE
Networking our LG washing machine

### ISSUE THREAD

**Issue by iteles** [dwyl/home#132]
OK, I know this sounds ridiculous 🙆‍♀️ 

But our LG washing machine has the ability to be wi-fi enabled, with certain functionality being controlled from the [LG ThinQ app](https://www.lg.com/us/discover/thinq/laundry).

Here are the features that I think we could intelligently/usefully use and would therefore warrant the connection to wifi:
+ **Push notifications when washing cycles have been completed** meaning there is no going to check if the cycle is complete yet or clothes forgotten in the washer - this would be useful for our caretaker in the short term and for residents in the longer term once we build this into our home app
+ **Energy monitoring** with reports on the energy consumption of the recently used cycles and monthly averages
+ **Download and program custom cycles** for types of washing specific to `home` uses
+ **Smart diagnosis** - if the machine breaks down, this allows for an initial remote diagnosis of the issue by LG engineers

You can also remotely start washing cycles if washing is already in the drum but I don't see a strong use case for this just yet (as you can easily program the machine to start at the desired time when you put the clothing in - this can be done on the machine itself.

The wi-fi module [is open source](http://opensource.lge.com/osList/list?m=Mc014&s=Sc016) so we could eventually build something ourselves but in the meantime the higher priority before we could do this would be to make sure our network is secure.

**Comment by nelsonic** [comment]
@iteles yeah, I'm not keen for any random **IoT** appliances to have access to our _internal_ network. 
It's increasingly looking like we are going to need (_at least_) ***three*** networks in the house:
1. Regular Wifi for members (_secure and behind a double-nat router so **no inbound TCP connections**_)
2. Secure network for security 
3. DMZ https://en.wikipedia.org/wiki/DMZ_(computing) for these kinds of untrusted IoT devices.

> Relates to: Networking [Epic] https://github.com/dwyl/home/issues/133

Furthermore, I don't think you want to have the Controller App on your "main" iPhone as it will send you annoying notifications when you _leave_ the house and the Housemaster takes over ... 

You've gotta love it when big established companies do "Open Source" by providing a Zip file of the code ... http://opensource.lge.com/osList/list?m=Mc014&s=Sc016
![image](https://user-images.githubusercontent.com/194400/87994576-37295a80-cae5-11ea-8b97-611ee27786c6.png)
![image](https://user-images.githubusercontent.com/194400/87994607-4f997500-cae5-11ea-9a6c-afc12d1ea4d1.png)
![image](https://user-images.githubusercontent.com/194400/87994626-617b1800-cae5-11ea-9fce-c6a5045a9b5c.png)
Totally misses the point ...  Good luck _doing_ anything with this code. There are _zero_ instructions.
The `README.txt` is just saying that they are using [**`FreeRTOS`**](https://www.freertos.org) Micro controller OS.
![image](https://user-images.githubusercontent.com/194400/87994845-dea68d00-cae5-11ea-8e9a-2ae9c8f06094.png)
No other instructions in the directory. Maybe someone else has written up a guide online. (_highly doubt it_).
